packages/dumb/src/modules/print/print-view.ts

```diff
@@ -0,0 +1,673 @@
+/*
```

Comment:
I don't think that this is a service and that it would be appropriate. I don't have a better naming in particular either, this one is not that bad... If you really mind it that bad, we can brainstorm it but I see little value in it

Reply:
I think this shows a poor design in the first place, if we end up with concepts we are unable to categorize 😬 Let's leave it like that for now but we should keep in mind that if we ever need to refactor the Print view (for another lib for instance), the whole design of classes etc should be changed to concepts we know and understand better